### DISCUSSION TITLE
ModuleNotFoundError: No module named '_hdfext'

### DISCUSSION THREAD

**Discussion by jiwon-j** [fhs/pyhdf#61] [Q&A]
hi,
im just trying to import 'from pyhdf.SD import SD, SDC'
but the error 'ModuleNotFoundError: No module named '_hdfext' happen 
other discussions, littlebit long time ago, say that i have to copy _hdfext file to pyhdf folder, but where can i found it?
is there any other way, I'd appreciate it if you let me know

thanks!

**Comment by CxWangJohnny** [comment]
I encontoured the same problem, if you solved it, please tell me how to fix it, ty!!
